### PR TITLE
Version bump to 1.3.4

### PR DESCRIPTION
**Pull request by lukecarbis**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 ## Changelog #
 
+### 1.3.4 - 2019-07-22 ###
+
+* New: Block Lab grew to level 1.3.4. Block Lab learned **Custom Categories**.
+* Tweak: **@phpbits** used Pull Request. All right! The **`block_lab_get_block_attributes`** filter was caught!
+* Tweak: **Template Loader** used Harden. **Template Loader**'s defense rose!
+* Tweak: Booted up a TM! It contained **Unit Tests**!
+* Fix: Wild **Missing Filter in Inspector Controls** bug appeared! Go! Bugfix!
+* Fix: Enemey **Mixed Up Inspector Controls** fainted! @kienstra gained 0902a06 EXP. Points!
```
Comment (kienstra) [inline CHANGELOG.md]: This is very minor, but maybe `Enemey` should be `Enemy`. Thanks for the points 😄

**Review by kienstra — DISMISSED** [review]
**Approved**

Hi @lukecarbis,
This looks good! It updates the version in the 3 files that are needed, like in previous [version bump PRs](https://github.com/getblocklab/block-lab/pull/311/files).

**Comment by RobStino** [comment]
Just testing and noticing an issue with newly created blocks.
Doing some digging...

**Review by RobStino — DISMISSED** [review]
Built a new block and tested all fields in both the editor and the inspector.
Looks great!

**Review by RobStino — APPROVED** [review]
👍

**Pull request merged**